.github/workflows/run-bats-tests.yml

```diff
@@ -7,16 +7,14 @@ on:
   pull_request:
     types:
       - opened
-      - closed
-      - synchronized
+      - synchronize
       - reopened
-      - ready_for_review
```

Comment:
we might keep this one, as far as I understand this one is triggered when PR is switched from "draft" to "ready for review".